File: public/phase-2-20160620022203/JAVASCRIPT/strings-problemo.js
// Name : Strings Problemo
// input:
// s : latin characters
// t : latin characters
// n : the number of possible replacements
// abw : each row corresponds to a b w for each possible replacement
var idx,answer,resulting_string;
var s = readline();
var t = readline();
var n = parseInt(readline(), 10);
var abw = new Array([]);
var abw_elements = "";
for(idx=0;idx<n;idx++){
	abw_elements = readline().split(' ');
	abw[idx] = new Array([]);
	for(jdx=0;jdx<3;jdx++){
		abw[idx][jdx] = abw_elements[jdx];
	}
}


// write your code here
// store your results in `answer`,`resulting_string`

// Dummy Data
answer = 21;
resulting_string = "uxyd";


// output
print(answer);
print(resulting_string);

File: public/phase-2-20160620022203/JAVA/strings-problemo.java
// Name : Strings Problemo
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// s : latin characters
		// t : latin characters
		// n : the number of possible replacements
		// abw : each row corresponds to a b w for each possible replacement
		int idx;
		int jdx;
		Scanner in = new Scanner(System.in);
		String s = in.nextLine();
		String t = in.nextLine();
		int n = Integer.parseInt(in.nextLine());
		char[][] abw = new char[n][3];
		String[] abw_elements = new String[100];
		for(idx=0;idx<n;idx++){
			abw_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<3;jdx++){
				abw[idx][jdx] = abw_elements[jdx];
			}
		}


		// write your code here
		// store your results in `answer`,`resulting_string`

		// output
		// Dummy Data
		int answer=21;
		String resulting_string="uxyd";

		System.out.println(answer);
		System.out.println(resulting_string);

	}
}